temperatureMappings.js

```diff
@@ -66,6 +66,7 @@ module.exports = {
     path: 'environment.inside.freezer.temperature'
   },
   'Exhaust Gas Temperature': {
-    path: 'propulsion.exhaust.temperature'
+    path: 'propulsion.0.exhaustTemperature',
```

Comment:
Afai can tell `pathWithIndex` overrides `path`?  So it should be removed not to confuse people (like me...).

Reply:
Maybe separately to get better release notes?